pkg/process/config/yaml_config.go

```diff
@@ -42,14 +40,10 @@ func (a *AgentConfig) LoadProcessYamlConfig(path string) error {
 		return err
 	}
 
-	URL, err := url.Parse(config.GetMainEndpoint("https://process.", key(ns, "process_dd_url")))
-	if err != nil {
-		return fmt.Errorf("error parsing process_dd_url: %s", err)
-	}
-	a.APIEndpoints[0].Endpoint = URL
-
-	if key := "api_key"; config.Datadog.IsSet(key) {
-		a.APIEndpoints[0].APIKey = config.SanitizeAPIKey(config.Datadog.GetString(key))
+	if eps, err := getAPIEndpoints(); err != nil {
```

Comment:
Nit: it would be easier to read if it were on multiple statements - you wouldn't need the `else`:
```
eps, err := getAPIEndpoints()
if err != nil {
    return err
}
a.APIEndpoints = eps
```

Reply:
Yeah, I was trying to be fancy 😅

I'll change it to what you suggested